components/Frame/UserNew.js

```diff
@@ -0,0 +1,121 @@
+import React, { Fragment } from 'react'
+import { css } from 'glamor'
+import { colors, mediaQueries, fontStyles } from '@project-r/styleguide'
+import { HEADER_HEIGHT, HEADER_HEIGHT_MOBILE } from '../constants'
+import { MdPersonOutline } from 'react-icons/md'
+import withT from '../../lib/withT'
+
+const BUTTON_SIZE = 32
+const BUTTON_SIZE_MOBILE = 24
+const BUTTON_PADDING = (HEADER_HEIGHT - BUTTON_SIZE) / 2
+const BUTTON_PADDING_MOBILE = (HEADER_HEIGHT_MOBILE - BUTTON_SIZE_MOBILE) / 2
+const ICON_SIZE = 26
+
+const PORTRAIT_HEIGHT_MOBILE = HEADER_HEIGHT_MOBILE - 2 * BUTTON_PADDING_MOBILE
+const PORTRAIT_HEIGHT = HEADER_HEIGHT - 2 * BUTTON_PADDING
+
+const getInitials = me =>
+  (me.name && me.name.trim()
+    ? me.name.split(' ').filter((n, i, all) => i === 0 || all.length - 1 === i)
+    : me.email
+        .split('@')[0]
+        .split(/\.|-|_/)
+        .slice(0, 2)
+  )
+    .slice(0, 2)
+    .filter(Boolean)
+    .map(s => s[0])
+    .join('')
+
+const User = ({ t, me, title, dark, backButton, ...props }) => {
+  const color = dark ? colors.negative.text : colors.text
+  return (
+    <div {...styles.user} {...props}>
```

Comment:
By spreading props here you assign the `onClick` handler to this div which does not have `role='button'` – role button and onClick should be on the same element. Generally it's better to assign explicits props that you support instead of forwarding rest props (creates uncertainty how this component is used – what other props are set on this div and why?).
```suggestion
    <div {...styles.user} {...props}>
```

Reply:
done